bayes_opt/acquisition.py

```diff
@@ -263,15 +274,14 @@ def _l_bfgs_b_minimize(self, acq: Callable, bounds: np.ndarray, n_x_seeds:int=10
 
         if min_acq is None:
             min_acq = np.inf
-            x_min = np.array([np.nan]*bounds.shape[0])
+            x_min = np.array([np.nan] * bounds.shape[0])
 
         # Clip output to make sure it lies within the bounds. Due to floating
         # point technicalities this is not always the case.
         return np.clip(x_min, bounds[:, 0], bounds[:, 1]), min_acq
 
-    def _update_params(self) -> None:
+    def _update_params(self) -> None:  # noqa: B027
```

Comment:
I'm not sure the way I tried to handle this case is correct. I want this function to
- a priori do nothing
- exist, so that I can call it during `.suggest`
- do something for specific functions (e.g. UBC)

Reply:
Initially, I decorated it with `abstractmethod`.
But in the notebook file, I saw that there was an object that inherited from this class.

I switched to public methods and tried to define them in the notebook file, 
but public methods need document.

It was difficult to write a precise description of this method, 
so I decided to ignore the `B027` rule for now. 

In my opinion, it would be better to define it as an abstract method.

Reply:
I thought about it again.
it would be better to switch to public method(not abstract method) and add docs.
(The reason for switching to a public method is that 
the user needs to know that they can override the method.)

Users can override this method if they want.
In any case, it seems to be the best way to make sure that it runs normally.

Reply:
Hi @phi-friday,

I also thought about it again.
I would suggest a different approach: Delete this function from the abstract base class, and retain it only for UBC/PoI/EI. Make it a public method. Call the function in the `.suggest` method for these 3, after the `super().suggest`. I think that's a clean solution -- what do you think?

Reply:
If the user is responsible for using `_update_params` (will be renamed),
that seems best.

Reply:
I've handled this in #498, [link](https://github.com/bayesian-optimization/BayesianOptimization/pull/498/commits/f31311601a6cee42c26219bc31b5ffe4331f3ceb).

Reply:
> I've handled this in #498, [link](https://github.com/bayesian-optimization/BayesianOptimization/pull/498/commits/f31311601a6cee42c26219bc31b5ffe4331f3ceb).

method names are also clearer than before, which I think is a good thing. 
I'll rebase and push again when #498  is merged.